migrate/page-index/page-index.js

```diff
@@ -74,6 +74,25 @@ module.exports = [
           },
         ],
       },
+      {
+        title: "Playbooks",
```

Comment:
Just want to double check with @VineethReddy02 that we are ok with this new section?

It will look like this:

![Screenshot 2023-11-23 at 14 59 38](https://github.com/timescale/docs/assets/5297542/a96376ed-7f21-47f8-b367-9304d8569d48)

Reply:
Probably would move multinode there as well. But in a separate PR

Reply:
Having playbooks as a tree lgtm!